core/ide/src/main/java/org/alice/stageide/sceneeditor/views/InstanceFactorySelectionPanel.java

```diff
@@ -310,15 +307,14 @@ protected void paintChildren(Graphics g) {
 
           Composite prevComposite = g2.getComposite();
 
```

Comment:
there's no reason to create nextComposite here instead of on line 325 when it is set??